packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -81,6 +81,8 @@ const hasDelegationCert = (certificates: Array<Cardano.Certificate> | undefined)
       case Cardano.CertificateType.StakeDelegation:
       case Cardano.CertificateType.StakeRegistration:
       case Cardano.CertificateType.StakeDeregistration:
+      case Cardano.CertificateType.Registration:
+      case Cardano.CertificateType.Unregistration:
```

Comment:
Same as previous comment, there are certificates that simultaneously alter the stake key registration state and do some additional work, those probably also need to be taken into account

Reply:
This was reworked in [last commit](https://github.com/input-output-hk/cardano-js-sdk/pull/1072/commits/6955bed9876017081e16fac3b6ccda01610b9bc9#diff-21aca0f26be89d933abdf7083bcc843606d20212c6a50847dfec6e966b09c33aR81).
The commits are cherry-picked one by one 😄 .

Reply:
Ah yes, I see it now 👍🏼